File: EuDecorator/Controllers/Dtos/IdentityResponse.cs
using System.Text.Json.Serialization;

namespace EuDecorator.Controllers.Dtos
{

    public class IdentityResponse
    {
        [JsonPropertyName("id")] public string Id { get; set; }

        /// <summary>
        /// TODO name should be plural
        /// </summary>
        [JsonPropertyName("verificationMethod")]
        public VerificationMethod[] VerificationMethods { get; set; }

        /// <summary>
        /// TODO name should be plural
        /// </summary>
        [JsonPropertyName("service")]
        public Service[] Services { get; set; }
    }
}

File: src/main/kotlin/nl/rijksoverheid/minienw/travelvalidation/validationservice/api/data/identity/IdentityResponse.kt
package nl.rijksoverheid.minienw.travelvalidation.validationservice.api.data.identity

import com.fasterxml.jackson.annotation.JsonProperty

class IdentityResponse
(
    @JsonProperty("id")
    val id: String,

    /**
    TODO name should be plural
     */
    @JsonProperty("verificationMethod")
    val verificationMethod :  Array<VerificationMethod>,

    /** TODO name should be plural*/
    @JsonProperty("service")
    val service : Array<Service>
)